packages/mgt-spfx/config/deploy-azure-storage.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this file needed?

Reply:
I can try removing it and see if it doesn't break the SPFx toolchain.